docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,307 @@
+:Number:BP000
+:Title: Blueprint specification and requirements
+:Author: Beraldo Leal <bleal@redhat.com>
+:Reviewers: 
+:Created: 29-Sep-2020
+:Type: Meta Blueprint
+:Status: Draft
+
+.. contents:: Table of Contents
+
+TL;DR
+*****
+
+Having better records of architectural decisions in our repository is a good
+way to socialize our technical decisions, improve the code review, avoid
+unnecessary code, and load balance the workload among other developers. We are
+already using the “blueprint” documents for a while, and still learning how to
+write them. Although we have a basic idea of what is a blueprint, some may not
+understand the concept because we are missing one meta-document that describes
+the blueprint's basic notions. This document is a blueprint to specify how we
+should write blueprints from now on.
+
+Motivation
+**********
+
+Depending on the project size, having very well defined and structured
+documents about the architecture decisions seems overkill at first glance, but
+could help any size projects, including Avocado, save time, make better
+decisions and improve the way we socialize those decisions.
+
+Today we have inside the Avocado project the good practice to submit an RFC to
+our mailing list or using a GitHub issue when we have a new idea. RFCs are a
+widespread way to disclose the architecture decisions, but they are just one
+part of the process. During the RFC phase, we argue in favor of a proposal, and
+we are mostly concerned about collecting feedback. After this phase, we could
+go one step forward and consolidate the discussion in Blueprints (sometimes
+called ADR - Architecture Decision Records). This could be the next step so we
+could better socialize our decisions for future readers. A very well defined
+and structured document has some advantages over an RFC, but it is not intended
+to replace it.
+
+With blueprints, we could not only, but mainly:
+
+  * Create better documents for future members and future reference.
```

Comment:
When using RFCs as email threads, there are no sections, headers, IMO, the RFCs as we use today are just thread discussions and are not focused on future review/reading. But if our RFCs has the same content, headers and goal of  BP, then yes, it could be.

Reply:
Alright, it is good to mention this in the same specificity in the blueprint itself. This makes it much clearer.